app/api/v2/resource/users.rb

```diff
@@ -68,12 +68,29 @@ def verify_otp!
                    type: String,
                    allow_blank: { value: false, message: 'resource.user.empty_topic' },
                    desc: 'Topic of user activity. Allowed: [all, password, session, otp]'
+          optional :time_from,
+                   type: { value: Integer, message: 'resource.user.non_integer_time_from' },
+                   allow_blank: { value: false, message: 'resource.user.empty_time_from' },
+                   desc: "An integer represents the seconds elapsed since Unix epoch."\
+                         "If set, only orders created after the time will be returned."
```

Comment:
`only orders` - what orders XD

Seems like just a copypaste problem.
Please, check all of the descs